rye/src/cli/tools.rs

```diff
@@ -40,11 +40,15 @@ pub fn execute(cmd: Args) -> Result<(), Error> {
 
 fn list_tools(cmd: ListCommand) -> Result<(), Error> {
     let mut tools = list_installed_tools()?.into_iter().collect::<Vec<_>>();
-    tools.sort();
+    tools.sort_by_key(|(tool, _)| tool.clone());
 
     for (tool, mut info) in tools {
         if cmd.version_show {
-            echo!("{} {}", style(tool).cyan(), style(info.version).cyan());
+            if let Some(ref venv) = info.venv_marker {
+                echo!("{} {} ({})", style(tool).cyan(), info.version, venv.python);
```

Comment:
I find it would be clearer to show the toolchain used by the tool, feel free to remove if it is not desired.